CONTRIBUTING.md

```diff
@@ -33,23 +33,25 @@
 
         git log `git describe --tags --abbrev=0 web-animations-js/master`..web-animations-next/master --pretty=format:"  * %s"
 
-3.  Specify the new version inside `package.json` (for NPM), for example:
+1.  Specify the new version inside `package.json` (for NPM), for example:
 
     ```js
       "version": "3.13.37",
     ```
 
-4.  Submit both changes to web-animations-next then follow the procedure to push from web-animations-next to web-animations-js.
+1.  Build the polyfill with `npm install && grunt` then update `README.md`'s Build Target Comparison with the current gzipped sizes.
```

Comment:
Not sure, we should confirm with Polymer animation team about this. The focus on size originated from their desire to use web-animations-next-lite.